src/machine/board_lgt92.go

```diff
@@ -0,0 +1,107 @@
+// +build lgt92
+
+package machine
+
+import (
+	"device/stm32"
+	"runtime/interrupt"
+)
+
+const (
+	LED1 = PA12
```

Comment:
Can you also add this?

```go
	LED = LED1
```

Many examples use `machine.LED` for the first (or only) LED available.

Reply:
Done